recipes/chafa/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# By installing from a release tar-ball (see meta.yaml), we avoid the
+# need for ./autogen.sh and additional dependencies
+
+export PKG_CONFIG_PATH="${BUILD_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH:-}"
```

Comment:
Why is this needed? You should not be pulling anything from $BUILD_PREFIX here.

Reply:
@eunos-1128 you added this line - were you seeing issues with pkgconfig?

Reply:
@peterjc 

I added the line according to https://github.com/conda-forge/staged-recipes/pull/28721#issuecomment-2608503329

> Setting the PKG_CONFIG_PATH env var to the path where Conda's freetype2.pc file can be found.

As I tried conda-build again without the line, however, I found it not required.

Please delete the line.